webapp/src/Service/DOMJudgeService.php

```diff
@@ -286,11 +287,14 @@ public function getUpdates(): array
     {
         $contest = $this->getCurrentContest();
 
-        $clarifications  = [];
-        $judgehosts      = [];
-        $rejudgings      = [];
-        $internal_errors = [];
-        $balloons        = [];
+        $clarifications           = [];
+        $judgehosts               = [];
+        $rejudgings               = [];
+        $internal_errors          = [];
+        $balloons                 = [];
+        $shadow_difference_count  = 0;
+        $external_contest_sources = [];
```

Comment:
These don't align with the one below.

Reply:
wait, only the bottom one doesnt align right?